src/scheduler/node_info.h

```diff
@@ -310,6 +310,7 @@ const char *node_state_to_str(node_info *ninfo);
  *
  */
 place *parse_placespec(char *place_str);
+int compare_place(place *pl1, place *pl2);
```

Comment:
Please add a function header for compare_place and compare_selspec like all other functions have.